Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -213,48 +242,34 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate {
 
 extension SearchViewController: UITableViewDataSource {
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = super.tableView(tableView, cellForRowAtIndexPath: indexPath)
-
-        if let currentSection = SearchListSection(rawValue: indexPath.section) {
-            switch currentSection {
-            case .SearchSuggestions:
-                cell.textLabel?.text = searchSuggestions[indexPath.row]
-                cell.detailTextLabel?.text = nil
-                cell.imageView?.image = nil
-                cell.isAccessibilityElement = false
-                cell.accessibilityLabel = nil
-            case .BookmarksAndHistory:
-                if let site = data[indexPath.row] as? Site {
-                    cell.textLabel?.text = site.title
-                    cell.detailTextLabel?.text = site.url
-                    if let img = site.icon? {
-                        let imgUrl = NSURL(string: img.url)
-                        cell.imageView?.sd_setImageWithURL(imgUrl, placeholderImage: self.profile.favicons.defaultIcon)
-                    } else {
-                        cell.imageView?.image = self.profile.favicons.defaultIcon
-                    }
-                    cell.isAccessibilityElement = false
-                    cell.accessibilityLabel = nil
+        switch SearchListSection(rawValue: indexPath.section)! {
```

Comment:
Why did you remove the `if let` here? If the `rawValue:` gets an invalid value, the app will crash. Better to not show anything then instead I think. I know it is an edge case but we should take advantage of type safety features. Even if 'it can never happen'.